src/js/components/Article.js

```diff
@@ -491,6 +513,11 @@ export default class Article extends Component {
       controls = this._renderControls();
     }
 
+    let pagination;
+    if (this.props.pagination && this.state.atBottom && 'row' === this.props.direction) {
```

Comment:
Related to issue https://github.com/grommet/grommet-estories/issues/303. I am working on a fix for this bug, which will deal with this issue at the same time